src/api/dirmonitor/polling.c

```diff
@@ -0,0 +1,123 @@
+#include <stdlib.h>
+#include <string.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/stat.h>
+#include <SDL.h>
+
+enum {
+  EVENT_MODIFIED
```

Comment:
Either add a `NO_EVENT` entry, or:
```suggestion
  EVENT_MODIFIED = 1
```
otherwise `monitor->fds[i].events & EVENT_MODIFIED` will always be 0.

Reply:
gone one further and replaced it with a define which tends to perform better within the compiler due to enum semantics
Also turned it into a future bitmask by setting it to `(1 << 0)`
All the code that sets, checks or removes events was also updated to do so bitwise